### DISCUSSION TITLE
Possible to disable auto `workspace` version prefix?

### DISCUSSION THREAD

**Discussion by mgenware** [pnpm/pnpm#4542] [Q&A] [answered]
I noticed when I do `pnpm i <a package in my workspace>`, pnpm automatically adds `"package": "workspace:^1.2.3"` to `package.json`. This works as intended for most workspace packages.

Is there any ways to opt out of this behavior? I have a package that I've published to npm and don't update quite often, using `"package": "^1.2.3"` (without the workspace prefix) can save us from rebuilding the package everytime we run `pnpm i`.

**Comment by zkochan** [comment, marked as answer]
Check the [save-workspace-protocol](https://pnpm.io/workspaces#save-workspace-protocol) setting and other settings from the workspace docs.
